data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```diff
@@ -54,9 +48,167 @@
         side=4
         team_name=north-west
         user_team_name= _ "teamname^Northwest"
-        canrecruit=yes
         controller=human
         gold=150
-        fog=yes
     [/side]
+
+    [options]
+        [slider]
+            id="bonus_income"
+            default=15
+            min=5
+            max=40
+            name= _ "Hill Bonus Income"
+            description= _ "Gold Per Turn To Leader On Hill Keep"
+        [/slider]
+        [slider]
+            id="chest_gold"
+            default=100
+            min=25
+            max=500
+            step=25
+            name= _ "Treasure Chest Gold"
+            description= _ "Gold Inside Hill Treasure Chest"
+        [/slider]
+    [/options]
+
+    [event]
+        name=prestart
+        [objectives]
+            [objective]
+                description= _ "Defeat enemy leaders"
+                condition=win
+            [/objective]
+            [note]
+                description= _ "A side with a leader holding the hill keep gets +$bonus_income gold income each turn"
```

Comment:
```suggestion
                description= _ "A side with a leader holding the hill keep gets +$bonus_income gold income each turn."
```

Reply:
So the objectives win/lose conditions use capitalization to start the sentence but no period, and then objectives notes use standard capitalization with a period?

Reply:
I think if it's a full sentence it should have a period. If it's a partial sentence (eg a phrase like "Death of the leader") then I think it doesn't need a period.